src/components/v2v3/V2V3Project/V2V3ProjectSettings/pages/EditCyclePage/PayoutsSection/PayoutsTable/HeaderRows.tsx

```diff
@@ -33,24 +38,28 @@ export function HeaderRows() {
 
   return (
     <>
-      <PayoutsTableCell className="flex flex-col items-start gap-0 md:flex-row md:gap-10">
+      <PayoutsTableCell className="flex flex-col items-start justify-between gap-0 md:flex-row md:gap-10">
         <div className="py-5">
           <div className="mb-2 text-lg font-medium">
             <Trans>Payout recipients</Trans>
           </div>
-          <div className="text-secondary text-sm">
-            <Trans>
-              Juicebox provides trustless payroll capabilities to run automated
-              payouts completely on-chain.{' '}
-              <ExternalLinkWithIcon
-                href={helpPagePath(`/user/project/#payouts`)}
-              >
-                <Trans>Learn more about payouts</Trans>
-              </ExternalLinkWithIcon>
-            </Trans>
-          </div>
+          {hideExplaination ? null : (
+            <div className="text-secondary text-sm">
+              <Trans>
+                Juicebox provides trustless payroll capabilities to run
+                automated payouts completely on-chain.{' '}
+                <ExternalLinkWithIcon
```

Comment:
**suggestion (non-blocking)** instead of a whoe new component, perhaps could just add a `withIcon` prop to ExternalLink